test/unit/check_modules_test.js

```diff
@@ -6,27 +6,42 @@ var expect = require('chai').expect
 var checkModules = require('../../lib/check_modules')
 
 describe('Checking modules', function () {
-  var fixture_path = path.resolve(__dirname, '..', 'fixtures')
+  var fixturePath = path.resolve(__dirname, '..', 'fixtures')
+  var fixtureName
+
+  function basePath () {
+    return path.join(fixturePath, fixtureName)
+  }
+
+  function packageInfo () {
+    return require(path.join(basePath(), 'package.json'))
+  }
+
+  function dependencyPath () {
+    return path.join(basePath(), 'node_modules')
+  }
+
+  function run () {
+    return checkModules(packageInfo(), dependencyPath(), true)
+  }
+
+  beforeEach(function () {
+    fixtureName = null
+  })
 
   describe('when they are valid', function () {
     it('does not throw an exception', function () {
-      var packageInfo = require('../fixtures/valid_dependencies/package.json')
-      checkModules(
-        packageInfo,
-        path.resolve(fixture_path, 'valid_dependencies', 'node_modules')
-      )
+      fixtureName = 'valid_dependencies'
+      run()
```

Comment:
Run wasn't immediately obvious as far as what it does. May consider renaming it to expose intent